packages/server/tests/previewnetAcceptance.env

```diff
@@ -20,6 +20,7 @@ SERVER_REQUEST_TIMEOUT_MS=60000
 MEMWATCH_ENABLED=false
 WRITE_SNAPSHOT_ON_MEMORY_LEAK=false
 TEST_INITIAL_ACCOUNT_STARTING_BALANCE=500
-HBAR_RATE_LIMIT_TINYBAR=5000000000# 50 HBARs
-HBAR_RATE_LIMIT_DURATION=80000# 80 seconds
+HBAR_RATE_LIMIT_TINYBAR=9000000000# 90 HBARs
+HBAR_RATE_LIMIT_DURATION=180000# 180 seconds
```

Comment:
Wouldn't the 180 seconds slow down the tests a lot? 🤔

Reply:
I don't believe so. This variable only affects the reset timestamp, and we’re not actually waiting on it during the tests. It shouldn’t impact the tests duration.